src/Misc/layoutbin/powershell/Start-AzpTask.ps1

```diff
@@ -0,0 +1,84 @@
+<# 
+A PowerShell script that is used to invoke a VSTS task script. This script is used by the VSTS task runner to invoke the task script.
+This script replaces some legacy stuff in PowerShell3Handler.cs and turns it into a dedicated signed script. 
+since it is parameterized it can be signed and trusted for WDAC and CLM.
+#>
+
+param ( 
+    [Parameter(mandatory = $true)]
+    [string]$Name,
```

Comment:
What does this param mean? Can we name this param more in a more self-descriptive way?